### ISSUE TITLE
[Task] Start EDAM concept ID from 1 instead of 0

### ISSUE THREAD

**Issue by tschaffter** [Sage-Bionetworks/sage-monorepo#2563]
### What product(s) is this story for?

OpenChallenges

### Description

This is to align with all the other IDs that are generated with AUTO INCREMENT.

References to existing EDAM IDs need to be updated too in `input_data_type.csv` and `challenges.csv` (`operation_id` property).

Depends on #2557

### Have you linked this story to a GitHub Project?

- [X] I have linked this story to a GitHub Project and set its metadata.

**Comment by tschaffter** [comment]
The head of the CSV data for input data types:

```
"id","class_id","preferred_label"
"0","http://edamontology.org/data_0005","Resource type"
"1","http://edamontology.org/data_0006","Data"
"2","http://edamontology.org/data_0007","Tool"
```

Yet the IDs start with 1 in the SQL table:

<img width="555" alt="image" src="https://github.com/Sage-Bionetworks/sage-monorepo/assets/3056480/ae2e9475-de9e-46f7-9c8c-d8be969f851a">

**Comment by tschaffter** [comment]
This ticket will be addressed by #2564

**Comment by vpchung** [comment]
> References to existing EDAM IDs need to be updated too in input_data_type.csv and challenges.csv (operation_id property).

This is already addressed in the OC data sheet.
